tools/pkg/files/build

```diff
@@ -3,7 +3,7 @@
 set -e
 
 VER=$1
-REVISION=${2:-1}
+REVISION=$2
```

Comment:
Do not use undefined variables. If we do not provide a default for `REVISION`, we should later check that it is defined and not empty or error with a meaningful message otherwise - the same way `VER` is handled.
It's way more convenient to just read a message, than to try to figure out why some script deeper down behaves erratically because of mangled input.